core/src/main/java/org/kiwix/kiwixmobile/core/di/components/ActivityComponent.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.core.di.components
+
+import android.app.Activity
+import dagger.BindsInstance
+import dagger.Subcomponent
+import org.kiwix.kiwixmobile.core.di.ActivityScope
+import org.kiwix.kiwixmobile.core.di.modules.ActivityModule
+import org.kiwix.kiwixmobile.core.di.modules.DialogModule
+
+@ActivityScope
+@Subcomponent(
+  modules = [
+    ActivityModule::class,
+    DialogModule::class]
+)
+interface ActivityComponent {
```

Comment:
I think this component is unused and can be deleted

Reply:
This will probably no longer be the case when you merge with my changes to SearchActivity

Reply:
@jhbiggs this is the only substantial thing left to do, merge develop into your branch and resolve the conflicts.